pyannote/audio/core/inference.py

```diff
@@ -458,8 +484,23 @@ def aggregate(
                 step=frames.step,
             )
 
-        masks = 1 - np.isnan(scores)
-        scores.data = np.nan_to_num(scores.data, copy=True, nan=0.0)
+        if not disk_store:
+            masks = 1 - np.isnan(scores)
+        else:
+            np_is_nan = disk_store.get_array(
+                "np_is_nan", scores.data.shape, np.bool
+            ).data
+            masks = disk_store.get_array("masks", scores.data.shape, np.int64).data
+
+            np.isnan(scores.data, out=np_is_nan)
+            np.subtract(1, np_is_nan, out=masks)
+
+            _s = scores.sliding_window
+            masks = SlidingWindowFeature(
+                masks, SlidingWindow(start=_s.start, duration=_s.duration, step=_s.step)
+            )
+
+        scores.data = np.nan_to_num(scores.data, copy=False, nan=0.0)
```

Comment:
I disabled making a copy here as it did not see necessary.

Reply:
I encountered the same problem, troubled for many days, debug the code, change here to solved.